pkg/policies/opa/rego/docker/docker_add/addinsteadofcopy.rego

```diff
@@ -0,0 +1,9 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[run.id]{
+	run := input.add[_]
+    config := run.config
+    
+    not contains(config, ".tar")
```

Comment:
value of `config` variable may or may not be in form of an array, need to create checks for both possibilities. please keep edge cases in mind while writing policy code

Reply:
> not contains(config, ".tar") was added twice

hey shreyas, first one is ".tar" and second one is ".tar.".  there's full stop after tar in second one

Reply:
Agreed but its much better if you check using a function.